subo/command/compute_deploy_core.go

```diff
@@ -129,11 +130,27 @@ func ComputeDeployCoreCommand() *cobra.Command {
 			util.LogStart("installing...")
 
 			if localInstall {
-				if _, err := util.Command.Run("docker-compose up -d"); err != nil {
-					return errors.Wrap(err, "🚫 failed to docker-compose up")
+				var compose string
+				if _, err := util.Command.Run("docker compose version 2>&1 >/dev/null"); err == nil {
+					// Use Compose v2 if we're positive we have it
+					compose = "docker compose"
+				} else if _, err := exec.LookPath("docker-compose"); err == nil {
+					// Fall back to legacy compose if available.
+					compose = "docker-compose"
+				} else {
+					// YOLO. Try Compose V2 anyway. Works with containerd/nerdctl.
+					// See: https://github.com/containerd/nerdctl/issues/1368
+					compose = "docker compose"
 				}
 
-				util.LogInfo("use `docker ps` and `docker-compose logs` to check deployment status")
+				command := fmt.Sprintf("%s up -d", compose)
+
+				if _, err := util.Command.Run(command); err != nil {
+					util.LogInfo("Is Docker Compose installed? https://docs.docker.com/compose/install/")
+					return errors.Wrapf(err, "🚫 failed to run `%s`", command)
+				}
+
+				util.LogInfo("use `docker ps` and `docker compose logs` to check deployment status")
```

Comment:
this line should also use whatever the `compose` variable is.
```suggestion
				util.LogInfo(fmt.Sprintf("use `docker ps` and `%s logs` to check deployment status", compose))
```

Reply:
Incorporated into 2413626bbb7b42a420943d6aaddb0ffcac23dfeb